src/Azure.Functions.ArtifactAssembler/ArtifactAssembler.cs

```diff
@@ -201,20 +201,8 @@ private async Task CreateVisualStudioCoreToolsAsync()
                 EnsureArtifactDirectoryExist(coreToolsHostArtifactDirPath);
                 await Task.Run(() => FileUtilities.CopyDirectory(coreToolsHostArtifactDirPath, consolidatedArtifactDirPath));
                 Directory.Delete(coreToolsHostArtifactDirPath, true);
-
-                // consolidatedArtifactDirPath now contains custom core-tools host, in-proc6 and in-proc8 sub directories. Create a zip file.
-                var zipPath = Path.Combine(customHostTargetArtifactDir, $"{consolidatedArtifactDirName}.zip");
-                await Task.Run(() => FileUtilities.CreateZipFile(consolidatedArtifactDirPath, zipPath));
-                Console.WriteLine($"Successfully created target runtime zip at: {zipPath}");
-
-                Directory.Delete(consolidatedArtifactDirPath, true);
             }
 
-            // Delete directories
-            Directory.Delete(_inProc6ExtractedRootDir, true);
```

Comment:
Previously we were deleting the extracted directory. With this change, we are not doing that anymore. That may cause disk space issues. correct?

Reply:
yeah I delete those directories after we zip in the consolidated pipeline. Here is an example of a run where we don't run into the disk space issue and managed to zip up the VS artifacts: https://dev.azure.com/azfunc/internal/_releaseProgress?_a=release-environment-logs&releaseId=193&environmentId=195